vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj

```diff
@@ -84,7 +84,7 @@
       we have a better solution for handling the absence of WinRT string and error DLLs on Win7.
       -->
       <AdditionalDependencies>WindowsApp_downlevel.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <DelayLoadDLLs>api-ms-win-core-winrt-error-l1-1-0.dll;api-ms-win-core-winrt-error-l1-1-1.dll;api-ms-win-core-winrt-string-l1-1-0.dll;ChakraCore.Debugger.DLL</DelayLoadDLLs>
```

Comment:
>DelayLoadDLLs [](start = 7, length = 13)

Should we add v8jsi.dll to DelayLoadDLLs?

Reply:
It comes in via the NuGet's .targets file (that's the recommended approach)

---
In reply to: [387964924](https://github.com/microsoft/react-native-windows/pull/4234#discussion_r387964924) [](ancestors = 387964924)